platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -228,7 +228,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (nullable MGLAnnotationImage *)mapView:(MGLMapView *)mapView imageForAnnotation:(id <MGLAnnotation>)annotation;
 
-- (CGFloat)mapView:(MGLMapView *)mapView alphaForShapeAnnotation:(MGLShape *)annotation __attribute__((deprecated("Use -mapView:strokeColorForShapeAnnotation: or -mapView:fillColorForPolygonAnnotation:.")));
+/**
+ Returns the alpha value to use when rendering a shape annotation.
+
+ A value of `0.0` results in a completely transparent shape. A value of `1.0`,
+ the default, results in a completely opaque shape.
```

Comment:
Updated the docs:

![screen shot 2017-01-13 at 3 30 58 pm](https://cloud.githubusercontent.com/assets/1198851/21944744/bdca754a-d9a5-11e6-9cc4-3ec2ff294e37.png)